### DISCUSSION TITLE
How to install on a Mac?

### DISCUSSION THREAD

**Discussion by Vollpflock** [tphakala/birdnet-go#23] [Q&A] [answered]
Hi Tomi
Are there any instructions on how to install it on a mac (M1)? I am not technically savvy
Best,
Lino

**Comment by tphakala** [comment, marked as answer]
Hi Lino,

I don't currently have pre-built binaries for mac as macOS requires programs to be cryptographically signed, that is something I have not yet got around to do. So unfortunately at the moment only way to install on mac is to compile from source code.

I will provide signed mac binaries at some point though!
